src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1635,17 +1640,21 @@ def _validate_vmss_create_load_balancer_or_app_gateway(cmd, namespace):
                 raise CLIError('usage error:{}'.format(err))
 
 
-def get_network_client(cli_ctx):
-    from azure.cli.core.profiles import ResourceType
-    from azure.cli.core.commands.client_factory import get_mgmt_service_client
-    return get_mgmt_service_client(cli_ctx, ResourceType.MGMT_NETWORK)
+def get_network_client(cli_ctx, balancer_type):
+    if balancer_type == 'application_gateways':
+        return import_aaz_by_profile(cli_ctx.cloud.profile, "network.application_gateway")
+    if balancer_type == 'load_balancers':
+        return import_aaz_by_profile(cli_ctx.cloud.profile, "network.lb")
 
 
 def get_network_lb(cli_ctx, resource_group_name, lb_name):
     from azure.core.exceptions import HttpResponseError
-    network_client = get_network_client(cli_ctx)
+    LBShow = import_aaz_by_profile(cli_ctx.cloud.profile, "network.lb").Show
```

Comment:
Why not use `get_network_client(cli_ctx, 'load_balancers').Show`?
Personally, I think it's more readable

Reply:
`get_network_client` is used to get client class in `azure-mgmt-network` sdk. However, `import_aaz_by_profile` can get `aaz` classes not only for `network`. You can get other modules as well. So it can be used to get vmss class like 
```python
    VMSSNicShow = import_aaz_by_profile(cli_ctx.cloud.profile, "vmss.nic").Show
```

Reply:
In fact, the `get_network_client` I want to refer to is the custom method defined in `_validators.py`
```Python
def get_network_client(cli_ctx, balancer_type):
    if balancer_type == 'application_gateways':
        return import_aaz_by_profile(cli_ctx.cloud.profile, "network.application_gateway")
    if balancer_type == 'load_balancers':
        return import_aaz_by_profile(cli_ctx.cloud.profile, "network.lb")
```